packages/react-select/src/components/Select/useSelectStyles.ts

```diff
@@ -12,36 +13,197 @@ export const selectClassNames: SlotClassNames<SelectSlots> = {
   icon: 'fui-Select__icon',
 };
 
-const useStyles = makeStyles({
-  wrapper: {
-    // TODO: add styles
+// TODO(sharing) use theme values once available
+const horizontalSpacing = {
+  xxs: '2px',
+  xs: '4px',
+  sNudge: '6px',
+  s: '8px',
+  mNudge: '10px',
+  m: '12px',
+};
+const contentSizes = {
+  // TODO: borrowed this from Input, should be in the theme somewhere?
+  body1: {
+    fontSize: tokens.fontSizeBase300,
+    lineHeight: tokens.lineHeightBase300,
+  },
+  caption1: {
+    fontSize: tokens.fontSizeBase200,
+    lineHeight: tokens.lineHeightBase200,
   },
-  select: {
-    // TODO: add styles
+  400: {
+    fontSize: tokens.fontSizeBase400,
+    lineHeight: tokens.lineHeightBase400,
   },
-  selectDisabled: {
-    // TODO: add styles
+};
+// TODO: borrowed this from Input (Select has the same size in design comps)
+// Should be in the theme somewhere?
+const fieldHeights = {
+  small: '24px',
+  medium: '32px',
+  large: '40px',
+};
+
+// TODO: borrowed from Input, also seems like animation should be included in the theme:
+const motionDurations = {
+  ultraFast: '0.05s',
+  normal: '0.2s',
+};
+const motionCurves = {
+  accelerateMid: 'cubic-bezier(0.7,0,1,0.5)',
+  decelerateMid: 'cubic-bezier(0.1,0.9,0.2,1)',
+};
+
+const useRootStyles = makeStyles({
+  base: {
+    alignItems: 'center',
+    boxSizing: 'border-box',
+    display: 'flex',
+    flexWrap: 'nowrap',
+    fontFamily: tokens.fontFamilyBase,
+    position: 'relative',
+
+    '*, *:before, *:after': {
+      boxSizing: 'border-box',
+    },
+
+    '&:after': {
+      backgroundImage: `linear-gradient(
+        0deg,
+        ${tokens.colorCompoundBrandStroke} 0%,
+        ${tokens.colorCompoundBrandStroke} 50%,
+        transparent 50%,
+        transparent 100%
+      )`,
+      ...shorthands.borderRadius(0, 0, tokens.borderRadiusMedium, tokens.borderRadiusMedium),
+      content: '""',
+      height: tokens.borderRadiusMedium,
```

Comment:
Do you mean to have a height value set to a border radius? (I don't really understand what is going on with this after element though :) )